<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="a_btn_add">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>720</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(25, 36, 106);
border:0px;</string>
   </property>
   <property name="text">
    <string>추가</string>
   </property>
  </widget>
  <widget class="QPushButton" name="a_btn_cancle">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>720</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
color: rgb(255, 255, 255);
background-color: rgb(25, 36, 106);
border:0px;</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_la_id">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>옷</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_la_season">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>계절</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_la_tem">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>380</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string>온도</string>
   </property>
  </widget>
  <widget class="QLabel" name="a_img_tem">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>110</y>
     <width>71</width>
     <height>460</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="a_img_clothes2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="a_img_clothes1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="a_te_name">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>150</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;나눔바른고딕 Light&quot;;
selection-background-color: rgb(44, 56, 97);
color: rgb(42, 61, 111);
background-color: rgb(197, 218, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLabel" name="a_img_clothes3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="a_img_clothes6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="a_img_clothes5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>430</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="a_img_clothes4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>141</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(217, 197, 255);
font: 25 20pt &quot;나눔바른고딕 Light&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="a_gb_season">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>300</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="a_rb_s_winter">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 12pt &quot;나눔바른고딕 Light&quot;;</string>
    </property>
    <property name="text">
     <string>겨울</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_s_spring_fall">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 12pt &quot;나눔바른고딕 Light&quot;;</string>
    </property>
    <property name="text">
     <string>봄/가을</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="a_gb_tem">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>450</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="a_rb_t4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 12pt &quot;나눔바른고딕 Light&quot;;</string>
    </property>
    <property name="text">
     <string>5℃~8℃   초겨울</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_t3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;나눔바른고딕 Light&quot;;</string>
    </property>
    <property name="text">
     <string>9℃~11℃   쌀쌀함</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_t5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 12pt &quot;나눔바른고딕 Light&quot;;</string>
    </property>
    <property name="text">
     <string>~4℃   한겨울 </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_t2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 12pt &quot;나눔바른고딕 Light&quot;;</string>
    </property>
    <property name="text">
     <string>12℃~16℃   일교차</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_t1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 25 12pt &quot;나눔바른고딕 Light&quot;;</string>
    </property>
    <property name="text">
     <string>17℃~19℃  봄/가을</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="a_gb_clothes">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>521</width>
     <height>631</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QRadioButton" name="a_rb_clothes4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;나눔바른고딕 Light&quot;;
</string>
    </property>
    <property name="text">
     <string>후드집업</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_clothes6">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>580</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;나눔바른고딕 Light&quot;;
</string>
    </property>
    <property name="text">
     <string>패딩</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_clothes2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;나눔바른고딕 Light&quot;;
</string>
    </property>
    <property name="text">
     <string>코트</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_clothes3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;나눔바른고딕 Light&quot;;
</string>
    </property>
    <property name="text">
     <string>후리스</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_clothes1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;나눔바른고딕 Light&quot;;
border-bottom-color: rgb(153, 255, 247);</string>
    </property>
    <property name="text">
     <string>가디건</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="a_rb_clothes5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>580</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;나눔바른고딕 Light&quot;;
</string>
    </property>
    <property name="text">
     <string>자켓</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>140</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>a_gb_clothes</zorder>
  <zorder>a_btn_add</zorder>
  <zorder>a_btn_cancle</zorder>
  <zorder>a_la_id</zorder>
  <zorder>a_la_season</zorder>
  <zorder>a_la_tem</zorder>
  <zorder>a_img_tem</zorder>
  <zorder>a_img_clothes2</zorder>
  <zorder>a_img_clothes1</zorder>
  <zorder>a_te_name</zorder>
  <zorder>a_img_clothes3</zorder>
  <zorder>a_img_clothes6</zorder>
  <zorder>a_img_clothes5</zorder>
  <zorder>a_img_clothes4</zorder>
  <zorder>a_gb_season</zorder>
  <zorder>a_gb_tem</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
